rpc/types/src/base.rs

```diff
@@ -0,0 +1,125 @@
+//! The base data that appear in many RPC request/responses.
+//!
+//! These are the common "headers" or "base" types that are
+//! [`flattened`](https://serde.rs/field-attrs.html#flatten)
+//! into many of Monero's RPC types.
+//!
+//! The `Access*` structs (e.g. [`AccessResponseBase`]
+//! are pseudo-deprecated structs for the RPC payment system, see:
+//!
+//! - <https://github.com/monero-project/monero/commit/2899379791b7542e4eb920b5d9d58cf232806937>
+//! - <https://github.com/monero-project/monero/issues/8722>
+//! - <https://github.com/monero-project/monero/pull/8843>
```

Comment:
This module defines the base types that are flattened into almost every response.

The most common being the `{ status: String, untrusted: bool }` one.

I implemented the RPC payment versions for now.